tox.ini

```diff
@@ -13,7 +13,7 @@ deps =
 deps =
     -rrequirements.txt
     pep8
-commands = {envbindir}/pep8 freetextresponse/
+commands = {envbindir}/pep8 --max-line-length=120 freetextresponse/
```

Comment:
Which lines necessitated this?

Reply:
I must have forgot it. Thanks @stvstnfrd!